src/engine/renderer/glsl_source/reflection_CB_fp.glsl

```diff
@@ -66,6 +68,18 @@ void	main()
 	// compute reflection ray
 	vec3 reflectionRay = reflect(viewDir, normal);
 
-	outputColor = textureCube(u_ColorMap, reflectionRay).rgba;
+	outputColor = textureCube(u_ColorMapCube, reflectionRay).rgba;
+
+	#if defined(r_showCubeProbes)
+		viewDir = normalize(var_Position);
+		outputColor = textureCube(u_ColorMapCube, viewDir);
+
+		/* HACK: use sign to know if there is a light or not, and
+		then if it will receive overbright multiplication or not. */
+		if ( u_InverseLightFactor > 0 )
+		{
+			outputColor.rgb *= u_InverseLightFactor;
```

Comment:
If there is a single condition, then you could simply set `u_InverseLightFactor` to `1` in C++ code?

Reply:
Right, fixed now.